frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -322,20 +322,25 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
         renderTile={renderTile}
         emptyStateInfo="No OperatorHub items are being shown due to the filters being applied."
       />
-      <Modal
-        show={!!detailsItem && showDetails}
-        onHide={closeOverlay}
-        bsSize="lg"
-        className="co-catalog-page__overlay right-side-modal-pf"
-      >
-        {detailsItem && (
-          <OperatorHubItemDetails
-            namespace={props.namespace}
-            item={detailsItem}
-            closeOverlay={closeOverlay}
-          />
-        )}
-      </Modal>
+      {detailsItem && (
+        <Modal
+          className="co-catalog-page__overlay co-catalog-page__overlay--right"
+          header={
+            <CatalogItemHeader
+              iconClass={detailsItem.iconClass}
+              iconImg={detailsItem.imgUrl}
+              title={detailsItem.name}
+              vendor={`${detailsItem.version} provided by ${detailsItem.provider}`}
+            />
+          }
+          isOpen={!!detailsItem && showDetails}
+          onClose={closeOverlay}
+          title={detailsItem.name}
+          width={'50%'}
```

Comment:
If the PatternFly doc is any indication, this is a problem at mobile, no?  
![Screen Shot 2019-12-12 at 1 52 23 PM](https://user-images.githubusercontent.com/895728/70740232-b7cde000-1ce6-11ea-8889-cfd5169dd113.png)

Our existing convention is to make the modal the (nearly the) full size of the viewport at mobile.
![localhost_9000_k8s_all-namespaces_pods(iPhone 5_SE)](https://user-images.githubusercontent.com/895728/70740329-e946ab80-1ce6-11ea-9561-388aced82354.png)